### ISSUE TITLE
SDK 2 Settings not available on api.js init

### ISSUE THREAD

**Issue by irritanterik** [athombv/homey#1821]
**What is your Homey version (Settings → System)?**
1.5.2

**What did you try to achieve?**
Port http app to SDK v2.
I read a setting on api.js initiation to determine the public argument (old working code: https://github.com/irritanterik/homey-http-request-actions/blob/master/api.js#L58)
`requires_authorization: (Homey.manager('settings').get('httpSettings') === undefined ? true : Homey.manager('settings').get('httpSettings').apiAuthorization),`

I tried several work-arounds and included `const Homey = require('homey')`, but Homey is not initiated. Homey.app is empty and Homey.ManagerSettings() is not a constructor. 

**Any other remarks**
How to read app settings and use them in api.js endpoint definitions?

**Code example of what i'm trying to achieve**
api.js
```
const Homey = require('homey')

var apiAuthorizationPublic = Homey.ManagerSettings.get('httpSettings') || false

module.exports = [
  {
    description: 'My endpoint',
    method: 'GET',
    path: '/:event',
    public: apiAuthorizationPublic,
    fn: function (callback, args) {
      callback(null, 'OK')
    }
  }
]
```
this results in stacktrace:
```
TypeError: Cannot read property 'httpSettings' of undefined
    at ManagerSettings.get (/opt/homey-client/system/manager/ManagerApps/bootstrap/sdk/v2/manager/settings.js:1:1139)
    at Object.<anonymous> (/api.js:4:52)
```

**Comment by irritanterik** [comment]
@jeroenvollenbrock Can you have a look at this one? Its my last impediment for the HTTP app SDK 2 rewrite

**Comment by irritanterik** [comment]
never mind, i't test again on 1.5.3 
`[SDKv2] Fixed a bug that caused App settings to be unavailable in the api.js file`

**Comment by irritanterik** [comment]
Confirmed fixed in 1.5.3 👍
